Source Linkage Files.Rproj

```diff
@@ -0,0 +1,13 @@
+Version: 1.0
```

Comment:
I'm not sure why this file was created. Could you just delete it please? or revert the commit which added it (would be better but slightly more work!)

Reply:
commit reverted